core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -488,7 +488,7 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
 
   test("resolveURIs with multiple paths") {
     def assertResolves(before: String, after: String): Unit = {
-      assume(before.split(",").length > 1)
+      assume(before.split(",").length >= 1)
```

Comment:
When running UtilsSuite, the following warning occurred:

Test Canceled: 1 was not greater than 1
org.scalatest.exceptions.TestCanceledException: 1 was not greater than 1
	at org.scalatest.Assertions$class.newTestCanceledException(Assertions.scala:511)
	at org.scalatest.FunSuite.newTestCanceledException(FunSuite.scala:1555)
	at org.scalatest.Assertions$AssertionsHelper.macroAssume(Assertions.scala:481)
	at org.apache.spark.util.UtilsSuite$$anonfun$19.assertResolves$2(UtilsSuite.scala:491)
	at org.apache.spark.util.UtilsSuite$$anonfun$19.apply$mcV$sp(UtilsSuite.scala:512)
	at org.apache.spark.util.UtilsSuite$$anonfun$19.apply(UtilsSuite.scala:489)
	at org.apache.spark.util.UtilsSuite$$anonfun$19.apply(UtilsSuite.scala:489)